### PR TITLE
Added 10 Logos

### PR DESCRIPTION
**Pull request by dbkonxepts**
Just a few popular logos retraced and refined for optimal PNG and SVG as instructed.
```
- Arla Food Inc.
- Bet9ja
- Dano Milk
- First City Monument Bank Ltd
- Milo
- Nestle
- Paga
- Sterling Bank Plc
- TeamApt
- Union Bank Nigeria
```

### CODE REVIEW

**Comment by fathermerry** [comment]
Love it! Thanks. Will review this asap.

**Comment by Odusanya** [comment]
DOYIN!! Thanks, man 💪🏽💪🏽💪🏽

File: logos/dano/dano.svg [hunk 1]
```diff
@@ -0,0 +1 @@
...tJK5Tqq3tyl22PdVY30Uhi6oWQAMxxTXGu/C+cTuqgfTgd1CPodCbGatl31hc66oHtjRtJUhBzm1GRa40rc0c56cwVo/tgM0F/7p/pKYAFgmZfsDutNj0jDLntrGKpO00Fa5TO3bSEUN/6IKeAP1rVE2bXGE8H+q6u4CG7ukfYJroPeT+3J6Ui7T8BfIAarRcEmH4qyAZyt+gmOf6dXISyyYoPVIDRRLauGDlJNKsKst8FguxgRlmMSTGFoAiPWzuCsnhb2e6YPvUudPsri3SJxDOaZ5QkJHay9GoTnM8xCuaoa8NJSgluvPiOvsz9ZxR+1PiUTwDhRpEGhcd5HncWnRk7/9z3R/C/BifxXTwTChve59TfjTEMn74IdmTx0/ExKp/JTjVf37gNL9F3XxzE2UXJIyHVts7Xh/HjkafKF0+dXwubityfp6ZnMa3P4ur9pecFE60lC8UNssJJn8vYiP5pf2DPDPRrTDdS9S10rrvk2Y5hG6R/9pHmke9INpQPQX4V5/EfC+qIhlkJIm4vqXGSnX1eubbnJkfm/OqWwyaisQVzjJPqbK9KnCc1G4ckuxpbERm/fbsS7PHT0+DtkGWZL+2PDxFFDynaho/2ZLbHHHaOvzDZ/Zod3S3yRcJ6VfwgfFztCCftLQTdnmwVFD3xZZm8alcb/+Hd5oFAXX4Vm3ZGSKa6bOEqnA68k5sxqKN/PS6p41EqzL5lCKVuU/KxGwRNySQnaWXfQEp/UQbwM9UOFKpq0q06Gn2sUhtdLPVHOM/ZGRAGyK6Doa+mTifhDXhscmeM+VagsrsYfB9HGbRHgPvrpqTXkMdlKfiL8Q2NvrJoeGOP8ie9vmFMK6xebfkReE/Rh6CaAeWsuHOqnM3hKqjGe/KUeLFgRkmSUnYZ4oLRyOsYdCokU1xRSto3nRVYydiuj423bmNaBGy78jjOgaI/1iJjTQKFDxOtsQyec05G67EFjDwbW8WlmZn0o3/p3aJTFwoE2j9oFXxZ4hmJ/ZCEJT56DADxUeMmIzoSWftwlfMnJP0FBOg2re1hr7aTNP7ero4+mrDxdRZH7+k+N9HNu7mC1jdebpNhRTcorEuFutZ0upDpnkBOXAIPZHvixV3RPpdaue8iOl94YrOSFETQCU20biDnZqOsffIGYp3bLG3NfhmWbGabPmE7wy9ATanzEArzHvZAtodsT2
```
Comment (Odusanya) [inline logos/dano/dano.svg]: Hey man, mad energy with this, but this part of the code has some png in it.
[image](https://user-images.githubusercontent.com/22182946/52441866-199b4780-2b22-11e9-96d4-69e51bb4b331.png)

Kindly take out the gradient image. I think the logo is fine if it's flat without the png
image

**Comment by fathermerry** [comment]
@Odusanya Kindly review again.

**Pull request merged**